v1.1.yml

```diff
@@ -1,6 +1,6 @@
 swagger: '2.0'
 info:
-  title: clever-api
+  title: Data API
```

Comment:
this also has events in it? Still want to call it this or should we separate out events?